server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -323,6 +325,10 @@ UNIT : U N I T;
 UNSTRING : U N S T R I N G;
 UP : U P;
 UPON : U P O N;
+USAGE : U S A G E;
```

Comment:
Please, do duplicate check, e.g. USE, USAGE (.. more) are already in  Db2SqlLexer.g4,